### DISCUSSION TITLE
Low battery = extremely choppy scroll

### DISCUSSION THREAD

**Discussion by wplit** [darkroomengineering/lenis#152] [General]
Small thing but just noticed that if you go below 10% battery, even on the latest Macbook M2 Max, Lenis scrolling is extremely choppy, worse than just native scroll. 

Would be useful if it was disabled when this happens, to return to normal scroll.

**Comment by clementroche** [comment]
When you go below 10% battery on MacOS it triggers low power mode, both Chrome and Safari are capping raf to 30 FPS in order to reduce consumption. I don't think that's something Lenis should take care internally, you can still add a condition to Lenis initialization.

```
if(!lowPowerMode) {
  new Lenis()
}
```
